portfolio/src/main/webapp/script.js

```diff
@@ -10,4 +10,20 @@
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
-// limitations under the License.
\ No newline at end of file
+// limitations under the License.
+
+const add = (datapoint) => {
+  const container = document.getElementById("data-container");
```

Comment:
[use single quotes](https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes), here and elsewhere
(it's confusing to have a different advice for Java and Javascript, but yolo)

Reply:
Updated with commit 64d662a

Reply:
`"data-container"` on line 16 is the last instance of double quotes remaining.

Reply:
"data-container" on line 16 is the last instance of double quotes remaining.